backend/package.json

```diff
@@ -38,10 +38,8 @@
     "fastify-plugin": "^1.6.1",
     "http-errors": "^1.7.3",
     "jsonwebtoken": "^8.5.1",
-    "jwks-rsa": "^1.7.0",
     "mongoose": "^5.9.6",
     "node-cache": "^5.1.0",
```

Comment:
We need both jwks-rsa and querystring, it's being used on authPlugin. Again, you changing code for no reason without proper context

Reply:
querystring is a Node.js module https://nodejs.org/api/querystring.html
I didn't find jwks-rsa in use, maybe removed that oauth method